content/en/docs/distributions/openshift/install-kubeflow.md

```diff
@@ -40,7 +40,7 @@ Minimal:
 Use the following steps to install Kubeflow 1.0 on OpenShift 4.x.
 
 1. Download the example "kfdef" for Kubeflow 1.3 on Openshift from [kubeflow/manifests/distributions/kfdef]
-(https://raw.githubusercontent.com/kubeflow/manifests/master/distributions/kfdef/kfctl_openshift_v1.3.0.yaml).
+(https://github.com/opendatahub-io/manifests/blob/v1.3-branch/distributions/kfdef/kfctl_openshift_v1.3.0.yaml).
```

Comment:
should it be: https://raw.githubusercontent.com/opendatahub-io/manifests/v1.3-branch/distributions/kfdef/kfctl_openshift_v1.3.0.yaml ?

Reply:
thanks @yhwang changed it to raw.